### PR TITLE
Update Lisk Elements index page

### LINKED ISSUE #1780: Update Lisk Elements index page
### Description

- The code snippets on the page need updates like version numbers, etc.
- The `Client` should be `client` in the NOTE.

#### Affected branch / URL
`docs-sdk`

#### Page location
`references/lisk-elements/index.adoc`

### PR DESCRIPTION
**Pull request by chris529**
Closes #1780

### CODE REVIEW

**Review by ishantiw — COMMENTED** [review]
If this update is related to 6.0.0 release in future then its correct but if its for beta or current release then it should `6.0.0-beta.1`

File: modules/ROOT/pages/references/lisk-elements/index.adoc [hunk 1]
```diff
@@ -60,7 +62,7 @@ Lisk Elements is a collection of JavaScript libraries, each tackling a certain t
 
 [NOTE]
 ====
-The `Client` package is specifically designed to be used in the browser and other frontends.
+The `client` package is specifically designed to be used in the browser and other frontends.
 It is a collection of the following packages:
 
 `apiClient`, `codec`, `cryptography`, `passphrase`, `transactions`, `utils`, `tree`, and `validator`.
```
Comment (TalhaMaliktz) [inline modules/ROOT/pages/references/lisk-elements/index.adoc]: I would mention the names of the packages verbatim.
```suggestion
The `lisk-client` package is specifically designed to be used in the browser and other frontends.
It is a collection of the following packages:

`lisk-api-client`, `lisk-codec`, `lisk-cryptography`, `lisk-passphrase`, `lisk-transactions`, `lisk-utils`, `lisk-tree`, and `lisk-validator`.
```

**Review by TalhaMaliktz — CHANGES_REQUESTED** [review]

**Comment by chris529** [comment]
@ishantiw Many thanks. However, this update is related to the v6.0.0 release. 👍

**Review by TalhaMaliktz — APPROVED** [review]
LGTM

**Review by ishantiw — APPROVED** [review]

**Pull request merged**